ios/kmbuild.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/sh
```

Comment:
I have renamed build.sh to kmbuild.sh and built a new wrapper build.sh that calls the oem product builds after completing the Keyman build.